pkgs/tools/archivers/peazip/default.nix

```diff
@@ -41,15 +40,25 @@ stdenv.mkDerivation rec {
 
   buildPhase = ''
     export HOME=$(mktemp -d)
-    cd dev
-    lazbuild --lazarusdir=${lazarus}/share/lazarus --widgetset=qt5 --build-all project_pea.lpi && [ -f pea ]
-    lazbuild --lazarusdir=${lazarus}/share/lazarus --widgetset=qt5 --build-all project_peach.lpi && [ -f peazip ]
-    cd ..
+
+    lazbuild --add-package packages/metadarkstyle-main/metadarkstyle.lpk
+
+    cd peazip-sources/dev
+
+    lazbuild --lazarusdir=${lazarus}/share/lazarus --widgetset=qt5 --build-all project_pea.lpi
+    [ -f pea ]
```

Comment:
Is this some species of "implicit abort"?

Reply:
it just checks if those files exist and exits with 1 if they don't. before it would only check if the previous command ran successfully but doesn't doesn't make sense. not sure if it makes sense to leave it in since the installPhase will fail anyway if those files don't exist.

Reply:
i don't remember why these tests were even added but i removed them because it'll fail in the installPhase anyway